app/views/remove_declaration.scala.html

```diff
@@ -0,0 +1,68 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import java.time.ZoneId
+@import java.time.Instant
+@import controllers.routes._
+@import controllers.util._
+@import forms.RemoveDraftDeclaration
+@import views.ViewDates
+@import uk.gov.hmrc.play.views.html._
+@import views.html.components.pagination
+@import views.components.inputs.RadioOption
+
+@this(main_template: views.html.main_template)
+
+
+@(declaration: ExportsDeclaration, form : Form[RemoveDraftDeclaration])(implicit request: Request[_], messages: Messages)
+
+@main_template(
+    title = messages("saved.declarations.remove.title"),
+    fullWidth = true
+) {
+
+    @components.back_link(routes.SavedDeclarationsController.displayDeclarations().url)
+
+    @components.error_summary(form.errors)
+
+    <h1>@messages("saved.declarations.remove.title")</h1>
+
+    @helper.form(RemoveSavedDeclarationsController.removeDeclaration(declaration.id.getOrElse("")), 'autoComplete -> "off") {
+        @helper.CSRF.formField
+        <table>
```

Comment:
This table doesn't need to be inside the form

Reply:
move the form inside the cell?

Reply:
Ignore!!! :man_facepalming: